### ISSUE TITLE
Add ability to subscribe to message

### ISSUE THREAD

**Issue by rosieks** [rebus-org/Rebus.ServiceProvider#6]
When I use `BuiltinHandlerActivator` I can easily subscribe to some events as part of my bootstrap code. I just call: `activator.Bus.Subscribe<SomeEvent>().Wait();`. It would be nice that also while using Rebus.ServiceProvider.

What do you think about extending `UseRebus` method to support such scenario:
    
    servicues.UseRebus(bus =>
    {
        bus.Subscribe<SomeEvent>().Wait();
    });

**Comment by mookid8000** [comment]
@trevorreeves how do you think this sounds?

**Comment by trreeves** [comment]
yeah, this sounds sensible, and pretty straight forward.

Note though, the extension method is on the `IServiceProvider`, not the `IServiceCollection`.  It might be appropriate to add a similar extension method for `IApplicationBuilder` as well, which is typically what is used in a `startup.cs`.

The solution will simply be:

```c#
public static IServiceProvider UseRebus(this IServiceProvider provider, Action<IBus> busAction)
{
   busAction(provider.GetRequiredService<IBus>());
   return provider;
}
```

Happy to add this if you like, plus I think a simple Startup.cs based sample would be appropriate (and modify the readme to show this), as that's the most common way people will setup Net Core applications.

**Comment by mookid8000** [comment]
@trevorreeves I don't have any experience with the ServiceProvider myself whatsoever – what do you say we give you commit rights to `Rebus.ServiceProvider` and then you can easily add the feature?

You can still make the commits on a separate branch and create a PR against master if you would like me to review the code.

**Comment by trreeves** [comment]
Yeah sure, that sounds good. I'd still prefer to do it via PRs as it's your
baby and I'm new to it.

Will have some time this week to address the issue.

**Comment by mookid8000** [comment]
Excellent 👍

**Comment by trreeves** [comment]
Resolved by PR https://github.com/rebus-org/Rebus.ServiceProvider/pull/8.

Available in 4.0.0-b06
